gensim/scripts/word2vec2tensor.py

```diff
@@ -63,16 +67,15 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
         True if input file in binary format.
 
     """
-    model = gensim.models.KeyedVectors.load_word2vec_format(word2vec_model_path, binary=binary)
+    model = gensim.models.KeyedVectors.load_word2vec_format(word2vec_model_path, binary=binary, datatype=np.float64)
```

Comment:
why `np.float64` instead of `np.float32` (that's default parameter)?

Reply:
For some reason, when the word2vec model is loaded, if we leave the default, using the test data file `word2vec_pre_kv_c` I run into parsing issues. For instance, for the word `the`, I get `array([-0.56110603, -1.97569799,  1.66395497, -1.23224604,  0.75475103,
        0.98576403,  2.26144099, -0.59829003, -0.47433099, -1.41610503], dtype=float32)` instead of `-0.561106 -1.975698 1.663955 -1.232246 0.754751 0.985764 2.261441 -0.598290 -0.474331 -1.416105`

Reply:
And what's a problem here? I still don't catch, sorry

Reply:
Sorry, should have been more clear. In the first dimension, as an example, the original word2vec model has `-0.561106` but for some reason, without the `np.float64` when it is read in, it displays `-0.56110603`, so the test fails equality. I'm not sure on the reason, though I assume something to do with the `load_word2vec_format` internals.

Reply:
what's assertion you mean? can you link concrete line of code?

Also, you can fix assertion (slightly relax "almost equal" condition), but anyway, you shouldn't change dtype here.